File: Demos/Dropwizard/src/main/java/com/groupdocs/ui/signature/entity/web/SignatureFileDescriptionEntity.java
package com.groupdocs.ui.signature.entity.web;

import com.groupdocs.ui.common.entity.web.FileDescriptionEntity;

/**
 * SignatureFileDescriptionEntity
 *
 * @author Aspose Pty Ltd
 */
public class SignatureFileDescriptionEntity extends FileDescriptionEntity {
    private String image;
    private String text;
    private String fontColor;

    public String getFontColor() {
        return fontColor;
    }

    public void setFontColor(String fontColor) {
        this.fontColor = fontColor;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    /**
     * Get encoded image Base64 string
     * @return image
     */
    public String getImage() {
        return image;
    }

    /**
     * Set encoded image Base64 string
     */
    public void setImage(String image) {
        this.image = image;
    }
}

File: Demos/Micronaut/src/main/kotlin/com/groupdocs/ui/model/SignatureFileDescriptionEntity.kt
package com.groupdocs.ui.model

open class SignatureFileDescriptionEntity(
     override val guid: String? = null,
     override val name: String? = null,
     override val docType: String? = null,
     override val directory: Boolean? = null,
     override val size: Long? = null,
     var image: String? = null,
     var text: String? = null,
     var fontColor: String? = null
) : FileDescriptionEntity()